python/semantic_kernel/connectors/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -34,175 +36,41 @@
-        service_id: Optional[str] = None,
-        api_key: Optional[str] = None,
-        ad_token: Optional[str] = None,
-        ad_token_provider: Optional[AsyncAzureADTokenProvider] = None,
-        default_headers: Optional[Mapping[str, str]] = None,
-        async_client: Optional[AsyncAzureOpenAI] = None,
-    ) -> None:
-        """
-        Initialize an AzureChatCompletion service.
-
-        Arguments:
-            deployment_name: The name of the Azure deployment. This value
-                will correspond to the custom name you chose for your deployment
-                when you deployed a model. This value can be found under
-                Resource Management > Deployments in the Azure portal or, alternatively,
-                under Management > Deployments in Azure OpenAI Studio.
-            base_url: The url of the Azure deployment. This value
-                can be found in the Keys & Endpoint section when examining
-                your resource from the Azure portal, the base_url consists of the endpoint,
-                followed by /openai/deployments/{deployment_name}/,
-                use endpoint if you only want to supply the endpoint.
-            endpoint: The endpoint of the Azure deployment. This value
-                can be found in the Keys & Endpoint section when examining
-                your resource from the Azure portal, the endpoint should end in openai.azure.com.
-                If both base_url and endpoint are supplied, base_url will be used.
-            api_key: The API key for the Azure deployment. This value can be
-                found in the Keys & Endpoint section when examining your resource in
-                the Azure portal. You can use either KEY1 or KEY2.
-            api_version: The API version to use. (Optional)
-                The default value is "2023-05-15".
-            ad_auth: Whether to use Azure Active Directory authentication. (Optional)
-                The default value is False.
+            service_id: The service ID for the Azure deployment. (Optional)
+            ad_token: The Azure Active Directory token. (Optional)
+            ad_token_provider: The Azure Active Directory token provider. (Optional)
             default_headers: The default headers mapping of string keys to
                 string values for HTTP requests. (Optional)
             async_client {Optional[AsyncAzureOpenAI]} -- An existing client to use. (Optional)
+            use_env_settings_file {bool} -- Use the environment settings file as a fallback to
         """
+        try:
+            azure_openai_settings = AzureOpenAISettings.create(use_env_settings_file=use_env_settings_file)
+        except ValidationError as e:
+            logger.error(f"Failed to validate AzureOpenAISettings: {e}")
+            raise ServiceInitializationError("Failed to validate AzureOpenAISettings") from e
+        base_url = azure_openai_settings.base_url
```

Comment:
Since we're handling the ability to manually set values via constructors, we do need to unpack all of them to do things like: `api_key = api_key or azure_openai_settings.api_key.get_secret_value() if azure_openai_settings.api_key else None`.